mininterface/tk_interface/adaptor.py

```diff
@@ -108,13 +108,22 @@ def run_dialog(self, form: TagDict, title: str = "", submit: bool | str = True)
 
         # Set the submit and exit settings
         if self.form.button:
-            tip, keysym = ("Enter", "<Return>")
+            tip = "Enter (when button focused)"
```

Comment:
I liked the behaviour as it was before, won't you put it back :) . Hitting Enter in any Entry does make sense to me. I admit it's bad to submit the form when hitting the file pick button but if you'd like to implement it, please do not change this behaviour. Probably, you'd need `.bind(..., add="+")` to tackle event bubbling or something.